CheckSysVinitOnSystemd.py

```diff
@@ -0,0 +1,73 @@
+#############################################################################
+# File          : CheckSysVinitOnSystemd.py
+# Package       : rpmlint
+# Author        : Werner Fink
+# Created on    : Tue Feb 21 17:34:50 2017
+# Purpose       : Check on systemd systems for required insserv package
+#############################################################################
+
+from Filter import addDetails, printError, printWarning
+import AbstractCheck
+import Config
+import string
+import os
+import stat
+import Pkg
+
+insserv_tag = 'insserv-is-required-on-systemd-based'
+etcinit_tag = 'etc-init-found-on-systemd-based'
+bootscr_tag = 'sysv-boot-script-found-on-systemd-based'
+
+#
+# Should be set in the global configuration file
+#
+#Config.setBadness(insserv_tag, 10000)
+#Config.setBadness(etcinit_tag, 10000)
+#Config.setBadness(bootscr_tag, 10000)
+
+if Config.info:
```

Comment:
normally this is at the end of the check section, can we move it downwards like in the other checks?